.github/workflows/build-and-push-containers.yml

```
@@ -0,0 +1,39 @@
+name: Build and push containers
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compile tag list
+        id: tags
+        run: |
+          TAG=${GITHUB_REF/refs\/tags\//}
```

```suggestion
          TAG=${GITHUB_REF#refs/tags/}
```

I'm sure I've already suggested this elsewhere.

---

Done.